notesScript.js

```diff
@@ -25,7 +36,46 @@ function createNoteElement(post)
         noteDiv.remove();
     });
 
+    //probably to seperate function
+
+    const commentButton = document.createElement('button');
+    commentButton.textContent = 'Pokaż komentarze';
+    commentButton.id = 'comment-button';
+
+    const commentContainer = document.createElement('div');
```

Comment:
lets move majortiy of this stuff to separate function and call it `displayCommentsForNote(noteId)`

this new function should  be reposnsible for fetching comments only for a one specific note and then displaying it on page

`createNoteElement` function should only create comment Button (lines 41-43) and assign `displayCommentsForNote` to onClick for this button